### PR TITLE
Update English doc of sermant agent

### LINKED ISSUE #329: Translate official website documents from Chinese to English
### What would you like to be added?

Translate official website documents from Chinese to English

### Why is this needed?

The English and Chinese documents on the official website are not synchronized.

### PR DESCRIPTION
**Pull request by daizhenyu**
What type of PR is this?

/Doc

What this PR does / why we need it?

sermant agent document translation

Which issue(s) this PR fixes:

Fixes #329

### CODE REVIEW

File: docs/en/document/user-guide/sermant-agent.md [hunk 1]
```diff
@@ -1,251 +1,441 @@
...
+If the log outputs as shown above, it indicates that Sermant Agent successfully read the startup command and began the installation. Open your browser and navigate to `http://localhost:8900`, where you should see a Sermant Agent instance. The expected result is shown in the image below: 
 
-**Loading order of plugins**
+<MyImage src="/docs-img/sermant-agent-agentmain-start.png"></MyImage>
+
+### Agent Uninstallation
+
+> Note: To avoid potential unpredictable exceptions when uninstalling service governance capabilities developed based on the premain startup method, Sermant Agent imposes restrictions on uninstallation. Only Sermant Agents started via the agentmain method support uninstallation; those started via the premain method do not support uninstallation.
+
+After [dynamically attaching the Agent](#agent Installation), you can uninstall Agent. Run `AgentLoader` again and pass the parameter to issue the uninstall command for Sermant Agent using `command=UNINSTALL-AGENT`:
```
Comment (hanbingleixue) [inline docs/en/document/user-guide/sermant-agent.md]: The format is incorrect.

File: docs/en/document/user-guide/sermant-agent.md [hunk 2]
```diff
@@ -1,251 +1,441 @@
...
+<MyImage src="/docs-img/sermant-agent-agentmain-uninstall-success.png"></MyImage>
 
-4. Bytecode enhancements at the same interception point for different plugins **take effect in the same order as plugin loading**.
+> Note: This capability can be implemented in the development mode by calling the `uninstall` interface provided by [AgentCoreEntrance](https://github.com/sermant-io/Sermant/blob/develop/sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/AgentCoreEntrance.java) in the sermant-agentcore-core module.
 
-### Parameter Configuration Options
+### Dynamic Plugin Installation
 
-The sermant project's properties configuration file and yaml configuration file for sermant-agent plugins support the following configuration options. Take `backend.nettyIp=127.0.0.1` in the configuration file for example:
+After [dynamically attaching the Agent](#agent Installation), you can dynamically install service governance plugins (provided the plugins support dynamic installation and uninstallation). Run `AgentLoader` again and pass the parameter to issue the dynamic plugin installation command using `command=INSTALL-PLUGINS:pluginA/pluginB`:
```
Comment (hanbingleixue) [inline docs/en/document/user-guide/sermant-agent.md]: Same as above

File: docs/en/document/user-guide/sermant-agent.md [hunk 3]
```diff
@@ -1,251 +1,441 @@
...
-Check to see if the log file for demo-application starts with the following:
+> Note: This script is only for use on **Linux systems**.
 
+#### Parameter Configuration
+
+- `-path=`：required parameter, followed by the **absolute path** of sermant-agent.jar.
+
+- `-pid=`：required parameter, followed by the PID of the host microservice, which can be found using commands like `ps -ef`.
+
+- `-command=`：required parameter, followed by the command to attach Sermant Agent. Supported commands can be found in the [Sermant Command Instructions](#sermant command instructions).
```
Comment (hanbingleixue) [inline docs/en/document/user-guide/sermant-agent.md]: Same as above

File: docs/en/document/user-guide/sermant-agent.md (hunk 1 above)
Comment (daizhenyu) [reply docs/en/document/user-guide/sermant-agent.md]: done

File: docs/en/document/user-guide/sermant-agent.md (hunk 2 above)
Comment (daizhenyu) [reply docs/en/document/user-guide/sermant-agent.md]: done

File: docs/en/document/user-guide/sermant-agent.md (hunk 3 above)
Comment (daizhenyu) [reply docs/en/document/user-guide/sermant-agent.md]: done

**Review by lilai23 — APPROVED** [review]

**Pull request merged**